runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/validator/ValidatorContext.java

```diff
@@ -19,14 +19,9 @@
 
 public interface ValidatorContext
 {
-    default ValidatorHandler attach(
+    default ValidatorHandler supplyHandler(
```

Comment:
I think we need to remove the `default` and `{ return null; }` as this must always be implemented.